### ISSUE TITLE
Table of Contents Data

### ISSUE THREAD

**Issue by hcayless** [distributed-text-services/specifications#112]
How should a client build or retrieve a table of contents? Which endpoint should provide the info for that ToC, and what should the info look like? Do we want to offer an expanded ToC with a single request? Related to #110 and #101.

**Comment by PonteIneptique** [comment]
This one is gonna get tricky. I feel like it should be in the <kbd>Navigation</kbd> endpoint, but how is the question. 

Do we want a `mode=curated` or `mode=ToC` which would allow people to provide a one page filtered <kbd>Navigation</kbd> object (where things are expanded by default) ? This might be seen as adding clutter though... But I see the point of this kind of functionality.

This might be tied to #167 to some extent: if we add an endpoint for `Annotations` or the like, then we could have an `AnnotationType: ToC` made of `Reference` trees.

**Comment by monotasker** [comment]
During the 2024 Durham RC Workshop we finalized the release candidate design of the Navigation endpoint. It does not include a dedicated TOC request, but a request can retrieve the entire citation tree of a document to a specified depth. This can easily be used to construct a TOC. 

We published the resolution of this issue during the tech committee meeting on 2024-03-08

commit https://github.com/distributed-text-services/specifications/commit/a0db8ca0f5b9ef2e208baae7cacd8e2b6108685b
release https://github.com/distributed-text-services/specifications/releases/tag/1-alpha1

This is an alpha release and we are looking for feedback!
